app/src/main/res/values-de/strings-de.xml

```diff
@@ -485,8 +481,305 @@
+	<string name="create_shortcut">Eine Abkürzung von der Navigationsleiste zum BZ-Level-Bildschirm.</string>
+	<string name="data_sync">Datensynchronisation</string>
+	<string name="default_liver_maximum_impact">Maximaler Standardeinfluss der Leber (0-1)</string>
+	<string name="delayed_but_more_stable">Verzögert aber stabiler bei verrauschten Werten. (nicht empfohlen)</string>
+	<string name="detect_motion_types">Sensoren benutzen um die Arten der Bewegungen zu erkennen</string>
+	<string name="dexcom_login">Dein Benutzername für Dexcoms Webseite</string>
+	<string name="dexcom_login_password">Dein Passwort für Dexcoms Webseite</string>
+	<string name="dexcom_password">Dexcom Accountpasswort</string>
+	<string name="dexcom_receiver_serial">10 Zeichen lange Seriennummer vom Dexcom Empfänger</string>
+	<string name="dexcom_test_mode_serial">10 Zeichen lange Testmodus-Seriennummer</string>
+	<string name="dexcom_user">Dexcom Accountbenutzername</string>
+	<string name="disable_log_transmitter_battery_warning">Deaktiviere die Warnung für einen niedrigen Transmitterbatteriestatus auf dem Startbildschirm. (Nur relevant für DIY-Empfänger.)</string>
+	<string name="display_calculations">Insulin-/Kohenhydratberechnungen anzeigen</string>
+	<string name="display_calculations_everytime">Insulin-/Kohenhydratberechnungen immer anzeigen.</string>
+	<string name="display_mathamatical_simulations">Mathematische Simulationen basierend auf den Profildaten und Kohlenhydrat-/Insulinlogs anzeigen.</string>
+	<string name="display_motion_types">Letzte Bewegungsdaten speichern und auf dem Hauptgraphen anzeigen.</string>
+	<string name="display_predictive_values">Vorhergesagte Werte anzeigen.</string>
+	<string name="enable_mongo_sync">Nightscout Mongo DB Synchronisation aktivieren</string>
+	<string name="enable_motion_tracking">Bewegungsverfolgung aktivieren</string>
+	<string name="enable_reminder_features">Reminder-Funktionen aktivieren</string>
+	<string name="enable_remote_logging">Remote-Logging aktivieren</string>
+	<string name="enable_telemetry">Telemetrie aktivieren</string>
+	<string name="enable_vehicle_mode">Fahrzeugmodus aktivieren</string>
+	<string name="engineering_mode">Entwicklermodus</string>
+	<string name="even_when_charging">Auch beim Laden</string>
+	<string name="experimental_calibration_plugin">Experimentelle sekundäre Kalibrierungsplugins</string>
+	<string name="experimental_rdfuino_support">Experimentelle Unterstützung für den RFduino von Tomasz Stachowicz</string>
+	<string name="experimental_transmitter_fpv_uav">Experimentelle Unterstützung für den das \'Transmitter\' Gerät von @FPV-UAV - nur Test</string>
+	<string name="extra_calibration_options">Extra Kalibrierungsoptionen</string>
+	<string name="extra_logging">Extra Loggingeinstellugen</string>
+	<string name="falling_threshold">Fall-Warngrenze</string>
+	<string name="flair_colors">Flair-Farben</string>
+	<string name="follower_chime_new">Neuer-Follower-Läuten</string>
```

Comment:
This one I neither understand in German nor is it long enough to do so in English without context.
The German translation would be "A new Follower - sound of bells"

Can you set a sound when a new follower joins @jamorham? In this case I'd use "Ton bei neuem Follower".